indico_toolkit/indico_wrapper/pred_manipulation.py

```diff
@@ -0,0 +1,115 @@
+from typing import List
+
+
+class ManipulatePreds:
+    """
+    Class to help expand prediction text and indexing via OCR data.
+    """
+
+    def __init__(self, ocr_tokens: List[dict], preds: List[dict]):
+        self.ocr_tokens = ocr_tokens
+        self.predictions = preds
+
+    def expand_predictions(self, pred_start: int, pred_end: int) -> dict:
+        """
+        Expand predictions and boundaries to match that of OCR data.
+
+        Args:
+            pred_start (int): Starting prediction index
+            pred_end (int): Ending prediction index
+        Returns:
+            dict: Returns expanded prediction dictionary
+        """
+        expanded_start, expanded_end = pred_start, pred_end
+        pred_index = None
+
+        # Find index value of bounded prediction
+        for index, pred in enumerate(self.predictions):
+            if pred["start"] == pred_start or pred["end"] == pred_end:
```

Comment:
I wonder here whether you should be looking for any overlap rather than an exact match on start/end-- I believe there is already a helper function for this somewhere